server/parsers/basic_input_output_parser.ts

```diff
@@ -13,19 +13,79 @@ const sanitize = (content: string) => {
   return DOMPurify.sanitize(content, { FORBID_TAGS: ['img'] }).replace(/!+\[/g, '[');
 };
 
+const isEveryItemString = (array?: unknown) =>
```

Comment:
```suggestion
const isEveryItemString = (array?: unknown): array is string[] =>
```

Maybe we can call it `isStringArray`?